src/pages/dashboard/DashboardPage.tsx

```diff
@@ -70,21 +75,53 @@ const DashboardPage = () => {
     gtfs_route_hour: item.gtfs_route_hour,
   }))
 
+  const datePickerErrorMessage = () => {
```

Comment:
suggestion:
```suggestion
  const datePickerErrorMessage = {
  	maxDate: 'נא לבחור תאריך שאינו עתידי',
  	minDate: 'נא לבחור תאריך בטווח מה-01.01.2010 עד היום',
  	invalidDate: 'נא להזין תאריך בפורמט תקין'
  }[datePickerError]
```

Reply:
It's throw error while trying to set the `datepickerErrorMessage`  as you suggest, because datePickerError is of DateValidationError type, and when trying to use `datePickerError` as index it throw error that some properties doesnt exist on `datePickerErrorMessage`.
for example:
`Property 'disableFuture' does not exist on type '{ maxDate: string; minDate: string; invalidDate: string; }'`
At the moment I can't find any optimal solution except to add the missing properties.
What to do prefer? Add the properties or use the switch-case statement?

Reply:
1. we can add the missing properties
2. we can make a function that transforms every unknown string to "unknown", and add an "unknown" property (`שגיאה לא ידועה`)